mbx/s2-4-suppprobs.mbx

```diff
@@ -1,12 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<section xml:id="sec_induction-suppprobs" xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
The xml:id here should probably still be sec_induction-suppprobs. Should be possible to update this PR by (from the ch2-supp branch) editing the line, committing, and pushing. That will automatically update the PR.